p2p/src/peer_manager/peerdb/mod.rs

```diff
@@ -71,9 +76,7 @@ impl<T: NetworkingService> From<&PeerContext<T>> for ConnectedPeer {
 }
 
 // TODO: Store available addresses in a binary heap (sorting by their availability).
-// TODO: Find a way to persist this data in some database for when the node is restarted
-// (banned, available, and at-least-once used should be restored)
-pub struct PeerDb<T: NetworkingService> {
+pub struct PeerDb<T: NetworkingService, P> {
```

Comment:
I'm very sorry for this dumb comment... but can you please use the letter S for store, like we do everywhere?

Reply:
Sure, updated.